### PR TITLE
Add Address Unit tests

### LINKED ISSUE #148: Insufficient Address Unit tests

### PR DESCRIPTION
**Pull request by reidenong**
fixes #148

### CODE REVIEW

File: src/test/java/seedu/address/model/person/AddressTest.java [hunk 1]
```diff
@@ -19,6 +19,20 @@ public void constructor_invalidAddress_throwsIllegalArgumentException() {
...
+        assertTrue(Address.isValidAddress("Apartment@456, Some-Place; City!"));
+    }
+
+    @Test
+    public void isValidAddress_longAddress() {
+        // Extremely long address (boundary test)
+        String longAddress = "A".repeat(1000); // A 1000-character string
+        assertTrue(Address.isValidAddress(longAddress)); // Assumes that the system allows very long addresses
+    }
+
```
Comment (Dominic-Khoo) [inline src/test/java/seedu/address/model/person/AddressTest.java:35]: Valid tests with appropriate boundary values.

**Review by Dominic-Khoo — APPROVED** [review]
LGTM!
